doc/dev/ffi.md

```diff
@@ -131,14 +131,21 @@ Thus `[init]` functions are run iff their module is imported, regardless of whet
 
 The initializer for module `A.B` is called `initialize_A_B` and will automatically initialize any imported modules.
 Module initializers are idempotent (when run with the same `builtin` flag), but not thread-safe.
+
+**Important for process-related functionality**: If your application needs to use process-related functions from libuv, you must call `lean_setup_libuv(argc, argv)` (which returns a potentially modified `argv` that must be used in place of the original) **before** calling `lean_initialize()` or `lean_initialize_runtime_module()`. This ensures that libuv's process handling capabilities are properly configured with the command-line arguments, which is essential for certain system-level operations that Lean's runtime may depend on.
```

Comment:
I think it would help here if you explained what counts as "process related functions"; perhaps this should explain that this calls `uv_setup_args` under the hood, as this is what the libuv functions refer to?